unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWCreatorController.cs

```diff
@@ -0,0 +1,249 @@
+using DCL;
+using DCL.Components;
+using DCL.Configuration;
+using DCL.Controllers;
+using DCL.Helpers;
+using DCL.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BIWCreatorController : BIWController
+{
+    public BIWModeController biwModeController;
+    public BIWFloorHandler biwFloorHandler;
+    public BuilderInWorldEntityHandler builderInWorldEntityHandler;
```

Comment:
If they are public only for the purpose to have dragged references in the editor, we should use the `SerializeField` attribute and make them private.